app/src/main/res/values/strings.xml

```diff
@@ -6,7 +6,9 @@
     <string name="app_name">Prepaid Balance</string>
     <string name="ussd_failed">Failed to request USSD code</string>
     <string name="unable_get_balance">Unable to get balance from response: %s</string>
-    <string name="permissions_required">Phone permissions required</string>
+    <string name="permissions_required_phone">Phone permissions required</string>
+    <string name="permissions_required_notification">Notification permission required</string>
+	<string name="permissions_required_phone_and_notification">Phone and Notification permissions required</string>
```

Comment:
This doesn't scale well. Maybe one string: `Additional permissions required`?

Reply:
I'd maybe keep phone permission info as is, cause it's an error and it should be clear which permission is missing..
But if notification permission is moved out of the worker and into the `onResume()` it can have its own message